dev_tools/auto_merge.py

```diff
@@ -62,7 +62,7 @@ def from_github(repo: GithubRepository,
 
         if response.status_code != 200:
             raise RuntimeError(
-                'Pull check failed. Code: {}. Content: {}.'.format(
+                'Pull check failed. Code: {}. Content: {!r}.'.format(
```

Comment:
Why is this change present? It's not related to what is described in the PR title or summary.

Reply:
fix pylint format warnings when running ./check/all